src/components/blankSlate/BlankSlate.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import * as _ from 'underscore';
+import { IBaseActionOptions } from '../actions/Action';
+import { Button } from '../button/Button';
+
+export interface IBlankSlateProps extends React.ClassAttributes<BlankSlate> {
+  title?: string;
+  description?: string;
+  buttons?: IBaseActionOptions[];
+  withModal?: boolean;
+}
+
+export class BlankSlate extends React.Component<IBlankSlateProps, any> {
```

Comment:
I know it is not that way in most of the other components, but you can use void as a type for the state (instead of any).